GPyOpt/objective_examples/experiments2d.py

```diff
@@ -13,36 +13,59 @@
 
 class function2d:
     '''
-    This is a benchmark of bi-dimensional functions interesting to optimize. 
-
+    This is a benchmark of bi-dimensional functions interesting to optimize.
     '''
-    
-    def plot(self):
+    def plot(self, ax=None, plt_type='contourf'):
         bounds = self.bounds
-        x1 = np.linspace(bounds[0][0], bounds[0][1], 100)
-        x2 = np.linspace(bounds[1][0], bounds[1][1], 100)
-        X1, X2 = np.meshgrid(x1, x2)
-        X = np.hstack((X1.reshape(100*100,1),X2.reshape(100*100,1)))
+        x1 = np.arange(*bounds[0], step=0.01)
```

Comment:
"*bounds[0]" is a neat trick, but let's probably revert to old code here. Old one is much more readable.